### PR TITLE
Connect in place: Prefix CSS animation names consistently

### PR DESCRIPTION
**Pull request by delawski**
This PR uses a consistent `jetpack-` prefix for CSS animation names. It's done in order to prevent potential name collisions since all the `@keyframes` live in the global scope.  

This addressed a feedback left on the following PR: https://github.com/Automattic/jetpack/pull/13553#pullrequestreview-294178418 

#### Testing instructions:

* Go to Jetpack dashboard
* Make sure you're disconnected from wordpress.com
* Initiate Jetpack connection and confirm that the 'Loading' text is pulsating and that the loading spinner below is animated

#### Proposed changelog entry for your changes:

* Add a consistent  `jetpack-` prefix to CSS animation names

### CODE REVIEW

**Review by simison — DISMISSED** [review]
I haven't tested the change but the code looks good.

Thanks for making it even better than it was before!

File: scss/atoms/_animations.scss [hunk 1]
```diff
@@ -2,7 +2,7 @@
 // Animation for miles!
```
Comment (simison) [inline scss/atoms/_animations.scss]: This comment is fairly confusing. :-D

Comment (delawski) [reply scss/atoms/_animations.scss]: I'll update it.

**Review by tyxla — COMMENTED** [review]
Noting that there are a bunch of more animations that are in `/_inc` and not in `/scss`. Are we planning to prefix those too?

**Comment by delawski** [comment]
@tyxla I'm hesitant to do so at this stage.

I guess it would make sense to move those animations to the `/scss/atoms/_animations.scss` at some point and rename them along the way. However, to perform such a migration well and in a safe manner, a substantial amount of time would have to be invested into a proper testing right now.

I think we should leave those animations as they are since we haven't noticed any issues so far. But let's keep that on our radar.

**Review by tyxla — APPROVED** [review]
🚢

**Comment by jeherve** [comment]
Cherry-picked to `branch-7.8` in 059eb733a757e7d3300ff74a9c89cb8889d796f8

**Pull request merged**